js_modules/dagit/src/runs/RunUtils.tsx

```diff
@@ -1,13 +1,44 @@
 import * as React from "react";
 import styled from "styled-components";
 import { Colors, Spinner } from "@blueprintjs/core";
+import gql from "graphql-tag";
+import { HandleStartExecutionFragment } from "./types/HandleStartExecutionFragment";
+import { showCustomAlert } from "../CustomAlertProvider";
 
 export type IRunStatus = "SUCCESS" | "NOT_STARTED" | "FAILURE" | "STARTED";
 
 export function titleForRun(run: { runId: string }) {
   return run.runId.split("-").shift();
 }
 
+export function handleStartExecutionResult(
+  pipelineName: string,
+  result: void | {
+    data?: { startPipelineExecution?: HandleStartExecutionFragment };
+  }
+) {
+  if (!result || !result.data || !result.data.startPipelineExecution) {
+    alert("No data was returned.");
```

Comment:
;)

Reply:
yeah this is a little lazy... one of those things that _shouldn't_ happen but I'll at least make it use the nice alert so it matches the other case.